src/output.rs

```diff
@@ -101,6 +112,92 @@ impl OutputStream for VerboseConsoleOutputStream {
     }
 }
 
+/// Struct that handles JSON console output formatting
+#[derive(Debug, Clone)]
+struct JsonConsoleOutputStream {}
+
+impl OutputStream for JsonConsoleOutputStream {
+    fn print_output(&self, panic_calls: &PanicCallsCollection) {
+        let stream = io::stdout();
+        for (i, trace) in panic_calls.calls.iter().enumerate() {
+            let json = json!({
+                "index" : i,
+                "pattern" : match trace.pattern.borrow().deref() {
+                    PanicPattern::Unrecognized => "unrecognized",
```

Comment:
Hmm, is this the best place to do this translation, or can we use the Display trait for this?

Reply:
I would say it's better to have all JSON-specific formatting explicit, especially since it allows the `Display` format to change without breaking the JSON rendering.

(I.e. you could view JSON as an "external" machine-readable format, that shouldn't change in backward-compatible ways.)

Reply:
Good point.